backend/de.metas.material/dispo-service/src/main/java/de/metas/material/dispo/service/candidatechange/handler/DemandCandiateHandler.java

```diff
@@ -194,7 +194,8 @@ private void fireSupplyRequiredEventIfQtyBelowZero(@NonNull final Candidate dema
 		Loggables.addLog("Quantity after demand applied: {}", availableQuantityAfterDemandWasApplied);
 
 		final BigDecimal requiredQty = computeRequiredQty(availableQuantityAfterDemandWasApplied, demandCandidateWithId.getMinMaxDescriptor());
-		if (requiredQty.signum() > 0)
+		final BigDecimal materialEventQty = demandCandidateWithId.getMaterialDescriptor().getQuantity();
```

Comment:
```suggestion
		// note: since this candidate might need to be handeled "lotForLot", we may fire the event even if requiredQty > 0
		final BigDecimal materialEventQty = demandCandidateWithId.getMaterialDescriptor().getQuantity();
```